initial-configuration/jenkins/jenkins-docker/jobs/Create Root Certs in TrustStore/config.xml

```diff
@@ -22,6 +23,15 @@
         <!--Import Certs to trust store-->
         keytool -import -keystore /usr/local/docker-config/wildfly/application.truststore -storepass password -noprompt -trustcacerts -alias letsencryptauthority1 -file isrgrootx1.der -storetype JKS
         keytool -import -keystore /usr/local/docker-config/wildfly/application.truststore -storepass password -noprompt -trustcacerts -alias letsencryptauthority2 -file lets-encrypt-r3.der -storetype JKS
+
+        <!--Remove existing psama trust store so it doesn't break-->
+        rm -rf /usr/local/docker-config/psama/application.truststore
+
+        curl https://letsencrypt.org/certs/isrgrootx1.der -o isrgrootx1.der
+        curl https://letsencrypt.org/certs/lets-encrypt-r3.der -o lets-encrypt-r3.der
+
+        keytool -import -keystore /usr/local/docker-config/psama/application.truststore -storepass password -noprompt -trustcacerts -alias letsencryptauthority1 -file isrgrootx1.der -storetype JKS
+        keytool -import -keystore /usr/local/docker-config/psama/application.truststore -storepass password -noprompt -trustcacerts -alias letsencryptauthority2 -file lets-encrypt-r3.der -storetype JKS
```

Comment:
How hard would it be to be super clever and pull the auth0 tenant from that env and add the corresponding cert to the trust store in the initial config pipeline?

Reply:
We can discuss in OH